Local Notification Not initialised properly.

**Sometimes app crashes due to local notification not being initialized properly.**


**To Reproduce**
Steps to reproduce the behavior:

1. Run the application
2. create a beacon 
3. when the timer is up, try joining the beacon
4. something as soon the timer is up it crases  or sometime after maps appears on the screen

The exception it throws is:
`Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'int java.lang.Integer.intValue()' on a null object reference flutter`


**Are you working on this issue? (Yes/No)** YES

--- a/lib/main.dart
+++ b/lib/main.dart
@@ -16,7 +16,7 @@ void main() async {
     DeviceOrientation.portraitDown,
   ]);
   setupLocator();
-  localNotif.initialize();
+  await localNotif.initialize();
   await hiveDb.init();
   runApp(
     OverlaySupport(

